build.gradle

```diff
@@ -156,6 +153,6 @@ ext.ociImageName   = 'local/' + ['bash', '-c', 'basename $(git config --get remo
 task buildImage(type: Exec) {
     group 'Build'
     description 'Builds an OCI image from a Dockerfile.'
-    commandLine("sh", "-c", "docker build -f docker/Dockerfile-local --build-arg spring_boot_jar=$bootJarPath -t $ociImageName:$gitShortCommit ."
+    commandLine("sh", "-c", "docker build -f docker/Dockerfile --build-arg spring_boot_jar=$bootJarPath -t $ociImageName:$gitShortCommit ."
```

Comment:
No palantir docker extension anymore?

Reply:
It was removed to converge between all repositories and to be able to quickly copy/paste the `docker build` command if needed.